service_container/3.3-di-changes.rst

```diff
@@ -81,9 +81,32 @@ what the file looks like in Symfony 4):
             </services>
         </container>
 
+    .. code-block:: php-fluent-di
+
+        // config/services.php
+        use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+        return function(ContainerConfigurator $configurator) {
+            $container = $configurator->services()
+                ->defaults()
+                ->autowire()
+                ->autoconfigure()
+                ->private();
+
+            $container->load('App\\', '../src/*')
+                ->exclude('../src/{Entity,Migrations,Tests}');
+
+            $container->load('App\\Controller\\', '../src/Controller')
+                ->tag('controller.service_arguments');
+        };
+
 This small bit of configuration contains a paradigm shift of how services
 are configured in Symfony.
 
+.. versionadded:: 3.4
+
+    PHP Fluent DI configuration wasn't available until version 3.4.
```

Comment:
PHP Fluent DI was introduced ja Symfony 3.4.

Reply:
I'm not sure if you are agreeing with me or you need to make a change..?

Reply:
IMO my sentence is more like in the other sf docs.

What do you think @javiereguiluz ?

Reply:
ah, I understand, you want me to change the content of my versionadded note to: `PHP Fluent DI was introduced in Symfony 3.4.`

Reply:
sounds good to me.

Reply:
Resolved.